docker_run.sh

```diff
@@ -3,9 +3,13 @@ set -o errexit
 
 source shared.sh
 
-docker run --env INPUT_JSON_URL='https://raw.githubusercontent.com/refinery-platform/shiny-docker-demo/v0.0.1/fixtures/fake-input.json' \
-           --name $CONTAINER_NAME \
+export INPUT_JSON="$(cat context/fixtures/food/fake-input.json)"
+echo 'INPUT_JSON:'
+echo $INPUT_JSON
+
+docker run --env INPUT_JSON \
```

Comment:
`INPUT_JSON_URL` Is what Refinery will be sending over to Tools, but I think is fine since I saw the support for it [above.](https://github.com/refinery-platform/intervene-refinery-docker/pull/6/files#diff-a613a674f8a7de5383710fe0175470a6R43)

Reply:
yeah, it should work either way, and this is simpler than starting a server just for the input.json